incidentresponse/wsgi.py

```diff
@@ -9,3 +9,42 @@
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'incidentresponse.settings.dev')
 
 application = get_wsgi_application()
+
+# dirty hackery is afoot here. This is modifying monzo's channel create functionality to be our own
+from response.slack.models.comms_channel import CommsChannelManager
+from urllib.parse import urljoin
+from django.urls import reverse
+from django.conf import settings
+import time
+from response.slack.client import SlackError
+import logging
+
+
+def create_incident_our_way(self, incident):
```

Comment:
Perhaps. The reason it is here is because the override is evaluated at the time the python interpreter loads the file into memory avoiding any chance of the default function ever being called. If we separated it we would need to orchestrate the execution in a similar mannor

Reply:
Ok - understood.